packages/manager/apps/pci-quota/src/components/Modal.component.tsx

```diff
@@ -0,0 +1,188 @@
+import { TServiceOption } from '@/api/data/service-option';
+import queryClient from '@/queryClient';
+
+export type TProps = {
+  type: 'support' | 'credit';
+  onConfirm: (formData: string) => void;
+  onCancel: () => void;
+  onClose: () => void;
+  isLoading: boolean;
+};
+
+type TState = {
+  serviceOption: TServiceOption;
+  description: string;
+};
+
+export const Modal = ({
+  type,
+  onConfirm,
+  onClose,
+  onCancel,
+  isLoading,
+}: TProps): JSX.Element => {
+  const { t, i18n } = useTranslation('quotas/increase');
+  const { projectId } = useParams();
+
+  const { data: project } = useProject(projectId);
+
+  const [state, setState] = useState<TState>({
+    serviceOption: undefined,
+    description: '',
+  });
+
+  const { data: issueTypes } = useGetIssueTypes(i18n.language);
+
+  useEffect(() => {
+    queryClient.invalidateQueries({
+      queryKey: ['issueTypes'],
+    });
+  }, [i18n.language]);
+
+  const inputLabel = useMemo(() => {
+    if (type === 'support' && Array.isArray(issueTypes)) {
+      const targetIssueType = issueTypes.find(({ id }) =>
+        ISSUE_TYPE_IDS.includes(id),
+      );
+
+      if (targetIssueType) {
+        return targetIssueType.fields.map(({ label }) => label).join('\n\n');
+      }
+    }
+    return '';
+  }, [issueTypes]);
+
+  const { data: serviceOptions } = useGetServiceOptions(projectId);
+
+  const selectCustomRenderer: OdsSelectCustomRenderer = {
+    item: (data: { text: string }) => {
+      return `<span>${data.text}</span>`;
+    },
+    option: (data: { text: string }) => {
+      return `<div style="display:flex;"><div style="flex-basis: 50%;">${data.text
+        .split('-')[0]
+        .trim()}</div><div style="flex-basis: 50%;text-align:right;">${data.text
+        .split('-')[1]
+        .trim()}</div></div>`;
+    },
+  };
+
+  return (
+    <OdsModal isOpen={true} onOdsClose={onClose}>
+      <div className="">
+        <div>
+          <OdsText preset="heading-4">
+            {t('pci_projects_project_quota_increase_title')}
+          </OdsText>
+        </div>
+        <div className="mt-8">
+          {type === 'credit' && (
+            <>
+              <OdsText preset="paragraph">
+                {t('pci_projects_project_quota_increase_buy_credits')}
+              </OdsText>
+              <div className="mt-4">
+                <OdsMessage color="warning" isDismissible={false}>
+                  <OdsText>
+                    {t('pci_projects_project_quota_increase_payment_info')}
+                  </OdsText>
+                </OdsMessage>
+              </div>
+            </>
+          )}
+          <OdsText preset="paragraph" className="mt-4">
+            <span
+              dangerouslySetInnerHTML={{
+                __html: t('pci_projects_project_quota_increase_content', {
+                  projectInfo: `<strong>${project?.description} ${project?.project_id}</strong>`,
+                }),
+              }}
```

Comment:
This shouldn't use dangerouslySetInnerHTML, you can use interpolation

Reply:
https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0

Reply:
I refactored it but then reverted it because the team told me we couldn't change the translation files (JSON), and doing the refactoring made me add a tag in the translations